src/util/file-filter.js

```diff
@@ -48,13 +49,16 @@ export class FileFilter {
     ignoreFiles = [],
     sourceDir,
     artifactsDir,
+    noDefaultIgnoreFiles = false,
   }: FileFilterOptions = {}) {
     sourceDir = path.resolve(sourceDir);
 
     this.filesToIgnore = [];
     this.sourceDir = sourceDir;
 
-    this.addToIgnoreList(baseIgnoredPatterns);
+    if (!noDefaultIgnoreFiles) {
```

Comment:
Not sure about this name for the option in the file filter, maybe can use something like `overrideDefaultIgnoreFiles`?

Aside, the filter has the option `baseIgnorePatterns`, that require an array. But I couldn't find an easy way to use in this case.